docs/release-notes.txt

```diff
@@ -6,6 +6,35 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.4
+^^^^^^^^^^^^^^
+
+**Release Date:** May 14, 2020
+
+- **Breaking Change:** Users are no longer automatically logged in as the "determined" user. Refer to the :ref:`users` documentation for more details.
```

Comment:
```suggestion
- **Breaking Change:** Users are no longer automatically logged in as the "determined" user. Refer to :ref:`users` for more details.
```
or
```suggestion
- **Breaking Change:** Users are no longer automatically logged in as the "determined" user. Refer to the :ref:`users <users>` documentation for more details.
```

If there's no explicit title text given, I'm of the opinion that the reference should be treated as a location, without the requirement that the title of the page fit nicely into the sentence. Same for the other links.

Reply:
Makes sense, good way of thinking about it.